### PR TITLE
Create cod1

### PR DESCRIPTION
**Pull request by Ajmensah**
Need corrections

### CODE REVIEW

**Comment by studenteng** [comment]
> Need corrections

yes you do

File: cod1 [hunk 1]
```diff
@@ -0,0 +1,6 @@
+This code will automate and reset client password every 60 days
+client will have get an email 
+that will prompt them to reset password
+password will have alot of requirements
+afafafdvss vvs
```
Comment (studenteng) [inline cod1]: remove this. It doesnt make sense

File: cod1 [hunk 2]
```diff
@@ -0,0 +1,6 @@
+This code will automate and reset client password every 60 days
+client will have get an email 
+that will prompt them to reset password
+password will have alot of requirements
+afafafdvss vvs 
+sfsdfsfgfgagtgfgfdgeffhhdfhsdhdghdghgfhfghfhgshsfhfghfghfghdbcbcbcvbgtr egregtgebbe ggtgeeget gegbcfbtrgrt gbbrtghbfbgg tgfbcbrthrbbbr  tghdbgrdbdb rtdhrgsethbbrgbbfgbnfgnfnfgnfdgncnxfnxcg gngfgnbg
```
Comment (studenteng) [inline cod1]: REmove this one too

**Review by studenteng — COMMENTED** [review]
Correct these

**Review by Ajmensah — COMMENTED** [review]
corrections are done

File: cod1 (hunk 1 above)
Comment (Ajmensah) [reply cod1]: done

File: cod1 (hunk 2 above)
Comment (Ajmensah) [reply cod1]: done

**Review by studenteng — APPROVED** [review]
good job

**Comment by Ajmensah** [comment]
great

**Pull request merged**